### ISSUE TITLE
[Feature Request] Ability to resize columns in table view.

### ISSUE THREAD

**Issue by SawkeeReemo** [WDaan/VueTorrent#1401]
Short and sweet: I love the rebuild, but would it be possible to be able to resize the column in table view at some point? Would be a welcome addition. Thank you for considering!

**Comment by tainewoo** [comment]
Or maybe to set the width of the "torrent title" column, by percentage?
Currently the torrent title column takes too much width percentage out of the total browser width.

If it's set in percentage, it can also fits different window scales if users switch window mode and maxmized mode and full screen mode for the browser.

**Comment by iamse7en** [comment]
+1 for this feature! Titles take way too much real estate for me in both desktop and mobile settings. One long title can widen the column far beyond what should be. I like how long titles skip to next line, but should do so much earlier in character count. Thanks for all the work on this theme, it's the very best!

**Comment by i0Z3R0** [comment]
Would love this feature. Titles take up way too much space with all the info in the title. A simple drag resize bar between the labels (title, size, ratio, etc.) would be great.

**Comment by puddingsoda** [comment]
+1 for me here, most of the time I want to quickly hide torrent names, and in qBittorrent it's very easy to change the title tab to 0% and mouse over if I want to know what a particular torrent is.

**Comment by theolaa** [comment]
I agree as well, the title takes up way too much width of each row (probably 40% just for title on my 1080p monitor), and honestly I usually only care about the first few dozen characters or so (40 maybe)? Resizable columns would be very welcome!

**Comment by swwrww** [comment]
Hi, just wanted to chime in and strongly support this request.

Being able to manually resize columns in Table view would significantly improve usability, especially on different screen sizes or when prioritizing specific fields like Name, Status, or ETA. Fixed-width columns can be quite limiting in everyday use.

It would also be great if column widths could be persisted across sessions (e.g. saved in localStorage or config), so users don’t have to keep adjusting them.

This enhancement would bring VueTorrent closer to the flexibility of traditional desktop clients, while keeping the clean UI.

Thanks for considering! 🙏

**Comment by kevingandrades** [comment]
+1 for this feature

**Comment by BlepBlep** [comment]
As someone who recently found out about this, I too would really love the ability to adjust (and keep static) the columns in table view. For now I've simply zoomed out enough to see all the columns but this is obviously not ideal lol

**Comment by kay-zhu** [comment]
"torrent title" takes up too much space. I hope it can be dragged to make it bigger or smaller. Thank you.
